app/assets/css/theme.css

```diff
@@ -86,6 +86,7 @@
   --green-3: #23ae89;
 
   --orange-1: #e86925;
+  --orange-2: #c77d23;
```

Comment:
can we use one of the `--ui-warning-` colors instead of defining a new one?

Reply:
fixed